src/AppRoutes.tsx

```diff
@@ -25,7 +25,7 @@ const AppRoutes: FC = () => {
       <Route exact path="/" component={Dashboard} />
       {isPublisher && <Route exact path="/new-publication" component={NewPublication} />}
       {isPublisher && <Route exact path="/my-publications" component={MyPublications} />}
-      {isAppAdmin && <Route exact path="/admin-institutions" component={InstututionPage} />}
+      {isAppAdmin && <Route exact path="/admin-institutions" component={AdministrateMemberInstututionPage} />}
```

Comment:
valgte å kalle det memberInstitution for å ikke blande med institusjons-komponentene.